src/models/Course.js

```diff
@@ -4,12 +4,12 @@ module.exports = (sequelize, DataTypes) => {
     shortcode: DataTypes.STRING,
     isUnlisted: {
       type: DataTypes.BOOLEAN,
-      allowNull: false,
+      allowNull: true,
```

Comment:
In retrospect, I think we should have a lot more `NOT NULL` constraints on columns. However, this is consistent with how other boolean attributes are treated, so we can file that away as a future enhancement 🙂